manifest.json

```diff
@@ -118,21 +118,21 @@
           },
           "platforms": [
             {
-              "dockerfile": "2.1/stretch/runtime",
+              "dockerfile": "2.1/stretch/amd64/runtime",
```

Comment:
The readme should also get updated to correctly reference the new Dockerfile locations.

Reply:
Updated.